XcodeServerSDKTests/RepositoryTests.swift

```diff
@@ -79,5 +79,35 @@ class RepositoryTests: XCTestCase {
         sshEnum = .LoggedInReadWrite
         XCTAssertEqual(sshEnum.toString(), "All logged in users can read and write")
     }
+    
+    // MARK: API Routes tests
+    func testGetRepositories() {
+        let expectation = self.expectationWithDescription("Get Repositories")
+        let server = self.getRecordingXcodeServer("get_repositories")
+        
+        server.getRepositories() { (repositories, error) in
+            XCTAssertNil(error, "Error should be nil")
+            XCTAssertNotNil(repositories, "Repositories shouldn't be nil")
+            
+            if let repos = repositories {
+                XCTAssertEqual(repos.count, 2, "There should be two repositories available")
+                
+                for (index, repo) in repos.enumerate() {
+                    XCTAssertEqual(repo.name, "Test\(index + 1)")
```

Comment:
Exactly, each of us can have a totally different Xcode Server setup and that's ok. The snapshot will depend on who created it, but since the `Cassette` ends up in git, it'll be very hard to lose it. I get your point, but I wouldn't worry about that too much. If we ever need to regenerate some response again for some reason we'll change it for that one run and then go back to the original config.

That's why it's nice that all `Cassette`s are independent.